docs/Deploying-Pega-on-PKS.md

```diff
@@ -54,23 +54,23 @@ The following account, resources, and application versions are required for use
 
 - kubectl – the Kubernetes command-line tool that you use to connect to and manage your Kubernetes resources.
 
-- PKS CLI - the PKS command-line tool that you use to manage and communicate with your PKS cluster.
+- Cloud Foundry CLI - the Cloud Foundry command-line tool that you use to manage and communicate with your TKGI cluster.
```

Comment:
Is Cloud Foundry correct?  That is a different technology.

Reply:
updated the references back to use pks cli...there's no more Cloud Foundry or cf CLI mentions.